fastlane_core/lib/fastlane_core/crash_reporter/crash_reporter.rb

```diff
@@ -62,7 +62,11 @@ def did_show_message?
 
         return did_show if did_show
 
-        File.write(path, '1')
+        begin
+          File.write(path, '1')
+        rescue
+          # don't show the user a message, since there is no functionality lost when this happens
```

Comment:
How would you feel about changing this to be something like `File.write(path, '1') rescue nil`?